src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: $main-text-color;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  background: url(../images/imac.jpeg) no-repeat;
+  background-size: 160px 134px;
+  background-position: top 32px center;
```

Comment:
we have a content image, it's not a background. So you need to create `img` tag at your html
```suggestion
```